src/spring-cloud/azext_spring_cloud/_help.py

```diff
@@ -462,7 +462,7 @@
     short-summary: Update Application Insights settings.
     examples:
         - name: Enable Application Insights and Java In-process Agent.
-          text: az spring-cloud app-insights update -n MyService -g MyResourceGroup --app-insights-key MyInstrumentationKey --sampling-rate 100
+          text: az spring-cloud app-insights update -n MyService -g MyResourceGroup --app-insights-key \"MyConnectionString\" --sampling-rate 100

```

Comment:
Since there is a semicolon within the connection_string, so double quotes the string, otherwise, will be ignored by the tokenizer or parser.
A sample connection_string: ```InstrumentationKey=9133e500-0432-48d9-b0d8-14915d4f1aab;IngestionEndpoint=https://eastus-6.in.applicationinsights.azure.com/```